fliplet-agent-install.js

```diff
@@ -60,7 +60,7 @@ try {
     // Listen for the "install" event, which indicates the
     // process is available as a service.
     svc.on('install',function(){
-      console.log('The agent has been installed successfully. You can see the logs in the Event Viewer under "Windows Logs" > "Application" for the source "flipletagent.exe"');
+      console.log(`The agent has been installed successfully. You can see the logs in the Event Viewer under "Windows Logs" > "Application" for the source "flipletagent${fileName}.exe"`);
```

Comment:
Just double checking that this format is correct, and doesn't need separators like `.` or `-`?

Reply:
Yes, this is correct. The filename produced by Windows is like "flipletagentsample.exe"